apps/publisher/src/App.tsx

```diff
@@ -35,6 +39,26 @@ function App() {
 
   const { startStreaming, stopStreaming, publisherState } = usePublisher(accessToken, streamId);
 
+  const {cameraList, microphoneList, selectedCamera, selectedMicrophone, setCameraHandler, setMicrophoneHandler, mediaStream} = useMediaDevices();
+
+  const video = useRef<HTMLVideoElement>(null)
+
+  useEffect(() => {
+    if (video.current && (mediaStream instanceof MediaStream)) {
+      video.current.srcObject = mediaStream;
+    }
+  }, [mediaStream])
+
+  const setCamHandler = (groupId: string) => {
```

Comment:
`setCamHander`, `setCameraHandler` so confused. Could be `onSelectCamera` , `setSelectedCamera`